masterlist.yaml

```diff
@@ -20951,6 +20951,15 @@ plugins:
         util: '[SSEEdit v4.0.2](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 12
 
+  - name: 'Caranthir Tower Reborn Expanded.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/62755' ]
+    dirty:
+      # version: 1
```

Comment:
The version number can be recorded in the commit message, so a comment isn't necessary:

```suggestion
```